### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by 9M2PJU**
Added MobaXterm - Enhanced terminal for Windows with X11 server, tabbed SSH client, network tools and much more

### CODE REVIEW

**Review by Shelldonna — APPROVED** [review]

**Review by eerr4 — APPROVED** [review]

**Review by Hatermakerbaby — COMMENTED** [review]
Good

**Review by Ghamz0727 — APPROVED** [review]

**Review by EfeErt67 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -215,6 +215,7 @@ Only main chapters:
 &nbsp;&nbsp;:small_orange_diamond: <a href="https://github.com/x-way/iptables-tracer"><b>iptables-tracer</b></a> - observe the path of packets through the iptables chains.<br>
 &nbsp;&nbsp;:small_orange_diamond: <a href="https://github.com/proabiral/inception"><b>inception</b></a> - a highly configurable tool to check for whatever you like against any number of hosts.<br>
 &nbsp;&nbsp;:small_orange_diamond: <a href="https://mremoteng.org/"><b>mRemoteNG</b></a> - a fork of mRemote, multi-tabbed PuTTy on steroids!<br>
+&nbsp;&nbsp;:small_orange_diamond: <a href="https://mobaxterm.mobatek.net/"><b>MobaXterm</b></a> - Enhanced terminal for Windows with X11 server, tabbed SSH client, network tools and much more<br>
 </p>
 
 ##### :black_small_square: Network (DNS)
```
Comment (Ghamz0727) [inline README.md:221]: README.md

**Review by RochanaRochana — COMMENTED** [review]
Hacking ⭐

**Comment by deecdub** [comment]
Nah

File: README.md (hunk 1 above)
Comment (deecdub) [reply README.md:221]: Expel the pass

Comment (Ghamz0727) [reply README.md:221]: ❤️

**Pull request closed without merging**